package.json

```diff
@@ -30,10 +30,10 @@
     "lint": "eslint ."
   },
   "devDependencies": {
-    "puppeteer": "^1.9.0",
     "eslint": "^6.1.0",
     "eslint-config-google": "^0.12.0",
-    "jest": "24.8.0",
-    "jest-each": "^24.5.0"
+    "jest": "^26.6.3",
+    "jest-each": "^24.5.0",
```

Comment:
should we use `jest-each: 26.6.2` to match `jest` version?

Reply:
*******